jobs/involuntary-dissolutions/involuntary_dissolutions.py

```diff
@@ -73,7 +78,78 @@ def shell_context():
     app.shell_context_processor(shell_context)
 
 
-def initiate_dissolution_process(app: Flask):  # pylint: disable=redefined-outer-name,too-many-locals
+def create_invountary_dissolution_filing(business_id: int):
+    """Create a filing entry to represent an involuntary dissolution filing."""
+    business = Business.find_by_internal_id(business_id)
+
+    filing = Filing()
+    filing.business_id = business.id
+    filing._filing_type = CoreFiling.FilingTypes.DISSOLUTION  # pylint: disable=protected-access
+    filing._filing_sub_type = DissolutionTypes.INVOLUNTARY  # pylint: disable=protected-access
+    filing._status = Filing.Status.PAID  # pylint: disable=protected-access
```

Comment:
Maybe no need to set status for filing? 💭 🤔  Just some observation, the [listener](https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/models/filing.py#L1033) will check and update the status before inserting. This is also shown in your testing results that the status of filing is DRAFT before filer processes it.

Reply:
Good point, updated!